py/server/deephaven/filters.py

```diff
@@ -80,21 +88,33 @@ def and_(filters: List[Filter]) -> Filter:
         filters (List[filter]): the component filters
 
     Returns:
-        a new Filter
+        a new and Filter
     """
     return Filter(j_filter=_JFilterAnd.of(*[f.j_filter for f in filters]))
 
 
 def not_(filter_: Filter) -> Filter:
-    """Creates a new filter that evaluates to true when the given filter evaluates to false.
+    """Creates a new filter that evaluates to the opposite of what filter_ evaluates to.
 
     Args:
         filter_ (Filter): the filter to negate with
 
     Returns:
-        a new Filter
+        a new not Filter
+    """
+    return filter_.not_()
+
+
+def is_null(col: str) -> Filter:
```

Comment:
For convenience, I would also consider adding `is_not_null` since it is so common, but you could convince me otherwise.

Reply:
I think that is the consensus after I polled others as well, adding it.